wiki/Beatmap/TAG/id.md

```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - multiplayer
+  - co-op
+---
+
+# TAG
+
+**TAG** adalah istilah yang digunakan untuk mendeskripsikan mode tim [multiplayer](/wiki/Multi) yang dinamakan [Tag Coop atau Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), di mana pemain-pemain mencoba untuk menyelesaikan sebuah [beatmap](/wiki/id/Beatmap) secara kooperatif. Jenis beatmap yang secara khusus didesain untuk mode osu!, disebut sebagai **Beatmap TAG/tingkat kesulitan**.
+
+Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan beralih di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga beatmap tersebut hampir tidak mungkin bisa dimainkan oleh hanya satu pemain.
+
+[Tingkat kesulitan](/wiki/id/Beatmap/Difficulty) TAG biasanya dilambangkan dengan penggunaan istilah "TAG" dalam nama kesulitan beatmap. Pada nama kesulitan "TAG" biasanya diikuti dengan angka, dengan khusus menunjukkan jumlah pemain yang harus bergabung untuk beatmap tersebut (misalnya, "TAG2" untuk dua pemain atau "TAG4" untuk empat pemain).
+
+Karena sifatnya, tingkat kesulitan TAG tidak diranked atau Loved, karena tingkat kesulitan TAG disengaja dirancang agar hampir mustahil dimainkan oleh hanya satu pemain.
```

Comment:
```suggestion
Karena sifatnya, tingkat kesulitan TAG memiliki status beatmap Unranked atau Loved, karena tingkat kesulitan TAG sengaja dirancang agar hampir mustahil dimainkan oleh satu pemain.
```

Reply:
`tidak diranked` sound like something common usage. changed to another way instead